pkg/streamingpromql/testdata/ours/functions.test

```diff
@@ -67,22 +67,65 @@ eval range from 0 to 6m step 1m floor(some_metric)
 clear
 
 load 1m
-  some_metric 0 1 2 3 _ _ {{schema:3 sum:4 count:4 buckets:[1 2 1]}} {{schema:3 sum:4 count:4 buckets:[1 2 1]}}
+  some_metric{foo="bar"} 0 1 2 3 _ _ {{schema:3 sum:4 count:4 buckets:[1 2 1]}} {{schema:3 sum:5 count:3 buckets:[2 5 4]}}
+  some_nhcb_metric{baz="bar"} {{schema:-53 sum:1 count:5 custom_values:[5 10] buckets:[1 4]}} {{schema:-53 sum:15 count:2 custom_values:[5 10] buckets:[0 2]}} {{schema:-53 sum:3 count:15 custom_values:[5 10] buckets:[7 8]}}
 
 eval range from 0 to 7m step 1m count_over_time(some_metric[3m])
-  {} 1 2 3 4 3 2 2 2
+  {foo="bar"} 1 2 3 4 3 2 2 2
 
 eval range from 0 to 7m step 1m count_over_time(some_metric[5s])
-  {} 1 1 1 1 _ _ 1 1
+  {foo="bar"} 1 1 1 1 _ _ 1 1
 
 eval range from 0 to 7m step 1m last_over_time(some_metric[3m])
-  some_metric 0 1 2 3 3 3 {{schema:3 sum:4 count:4 buckets:[1 2 1]}} {{schema:3 sum:4 count:4 buckets:[1 2 1]}}
+  some_metric{foo="bar"} 0 1 2 3 3 3 {{schema:3 sum:4 count:4 buckets:[1 2 1]}} {{schema:3 sum:5 count:3 buckets:[2 5 4]}}
 
 eval range from 0 to 7m step 1m last_over_time(some_metric[5s])
-  some_metric 0 1 2 3 _ _ {{schema:3 sum:4 count:4 buckets:[1 2 1]}} {{schema:3 sum:4 count:4 buckets:[1 2 1]}}
+  some_metric{foo="bar"} 0 1 2 3 _ _ {{schema:3 sum:4 count:4 buckets:[1 2 1]}} {{schema:3 sum:5 count:3 buckets:[2 5 4]}}
 
 eval range from 0 to 7m step 1m present_over_time(some_metric[3m])
-  {} 1 1 1 1 1 1 1 1
+  {foo="bar"} 1 1 1 1 1 1 1 1
 
 eval range from 0 to 7m step 1m present_over_time(some_metric[5s])
-  {} 1 1 1 1 _ _ 1 1
+  {foo="bar"} 1 1 1 1 _ _ 1 1
+
+eval range from 0 to 7m step 1m min_over_time(some_metric[3m])
+  {foo="bar"} 0 0 0 0 1 2 3 _
+
+eval range from 0 to 7m step 1m min_over_time(some_metric[5s])
+  {foo="bar"} 0 1 2 3 _ _ _ _
+
+eval range from 0 to 7m step 1m max_over_time(some_metric[3m])
+  {foo="bar"} 0 1 2 3 3 3 3 _
+
+eval range from 0 to 7m step 1m max_over_time(some_metric[5s])
+  {foo="bar"} 0 1 2 3 _ _ _ _
```

Comment:
We should check each of the above `_over_time` functions against native histograms even if they don't return to ensure consistency with promql.

Reply:
Not sure I follow sorry - this test is running the `_over_time` functions over a series with native histograms.

Reply:
Sorry, I think I saw `some_metric` and `some_nhcb_metric` above and thought we should add these for `some_nhcb_metric`. But yes, this does operate over NH's so lgtm, thanks.